### PR TITLE
adding photo to html2

### PR DESCRIPTION
**Pull request by AlexTurning**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,21 @@ <h2 style="color: black;"><span>PiRong</span> Koh</h2>
                             </figure>
                         </div>
                     </div>
+					  <div class="col-xs-4 col-sm-3 col-md-3 col-lg-3">
```
Comment (tradzik) [inline index.html]: Please improve tabulation in this line.

File: _data/students.yml [hunk 2]
```diff
@@ -130,3 +130,11 @@
...
   image: ../images/KohPR.jpg
   bio: 14, Student, Interested in Python, C++
+  
+- name: Saurabh Singh
+  email: iemailsaurabhsingh@gmail.com
+  github: AlexTurning
+  facebook: http://www.facebook.com/meetsaurabh.s 
+  image: ../images/Sa.jpg
+  bio: 16,Indian,keen learner,keep learning
+
```
Comment (tradzik) [inline _data/students.yml]: Please remove empty line at the end of the file.

**Review by tradzik — CHANGES_REQUESTED** [review]
Please [squash](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) your commits into one and rebase to make this PR easily mergable. Thanks.

**Comment by hemantjadon** [comment]
@AlexTurning please also resolve the merge conflicts

**Comment by jigyasa-grover** [comment]
@AlexTurning As @tradzik & @hemantjadon pointed, pls do the following:

* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Resolve merge conflicts, this [How To Manually Fix Git Merge Conflicts (video)](https://www.youtube.com/watch?v=g8BRcB9NLp4) might be helpful. 

Cheers !

**Comment by jigyasa-grover** [comment]
Wrt [rules in CONTRIBUTING.md](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md), closing this.

**Pull request closed without merging**